superset-frontend/src/common/components/index.tsx

```diff
@@ -34,6 +34,7 @@ export {
   Button,
   Card,
   Checkbox,
+  Collapse,
```

Comment:
This is the one I'm hesitant to export, if we're trying to encourage adoption of the custom one.

Reply:
Fixed.